Sources/NIOSSL/TLSConfiguration.swift

```diff
@@ -142,6 +191,11 @@ public struct TLSConfiguration {
     /// The pre-TLS1.3 cipher suites supported by this handler. This uses the OpenSSL cipher string format.
     /// TLS 1.3 cipher suites cannot be configured.
     public var cipherSuites: String
+    
+    
+    public var verifySignatureAlgorithms : [SignatureAlgorithm]
+    
+    public var signSignatureAlgorithms : [SignatureAlgorithm]
```

Comment:
Three notes here: firstly, SwiftNIO style is to have no leading space before the colon, i.e. `signSignatureAlgorithm: [SignatureAlgorithm]`.

Secondly, we should add some doc comments to these.

Third, let’s change `signSignatureAlgorithms` to `signingSignatureAlgorithms`.